build_test/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 0.8.0-dev
+
+- `InMemoryAssetReader`, `MultiAssetReader`, `StubAssetReader` and
+  `PackageAssetReader` now implement the `MultiPackageAssetReader` interface.
+- `testBuilder` now supports `Builder`s that call `findAssets` in non-root
+  packages. 
```

Comment:
[nit] Indentation seems off here.

Reply:
it just looks weird with the quotes

Reply:
Looks fine to me. I think the quote is just throwing it off visually.